src/features/eye/eye_analyzer.cc

```diff
@@ -0,0 +1,92 @@
+#include "src/features/eye/eye_analyzer.h"
+
+#include "mediapipe/framework/port/opencv_core_inc.h"
+#include "mediapipe/framework/port/opencv_imgproc_inc.h"
+
+EyeAnalyzer::EyeAnalyzer(int img_width, int img_height)
+    : GenericAnalyzer(img_width, img_height) {}
+
+EyeAnalyzer::EyeAnalyzer(mediapipe::NormalizedLandmarkList landmarks,
+            int img_width, int img_height) 
+                : GenericAnalyzer(landmarks, img_width, img_height) {}   
+
+double EyeAnalyzer::GetEyeInnerArea() {
+    return eye_area_;
+}
+
+double EyeAnalyzer::GetEyebrow() {
+    return eyebrow_anchor_dist_sum_;
+}
+
+void EyeAnalyzer::UpdateEyeInnerArea() {
+    GenerateEyeContours();
+    eye_area_ = CalculateEyesContoursArea();
+}
+
+double EyeAnalyzer::CalculateEyesContoursArea() {
+    double right_eye_area = cv::contourArea(eye_right_contour_);
+    double left_eye_area  = cv::contourArea(eye_left_contour_);
+
+    return right_eye_area + left_eye_area;
+}
+
+void EyeAnalyzer::GenerateEyeContours() {
+    mediapipe::NormalizedLandmark landmark;
+    int eye_right_lmark, eye_left_lmark, eye_area_;
+    int lmarks_amount = sizeof(EYE_RIGHT_INNER_LMARKS) / sizeof(int);
+    cv::Point lmark_cv_point;
+
+    for (int i=0; i < lmarks_amount; i++) {
+         
+        eye_right_lmark = EYE_RIGHT_INNER_LMARKS[i];
+        landmark = landmarks_.landmark(eye_right_lmark);
+        lmark_cv_point = CvtNormIntoCvPoint(landmark);
+        eye_right_contour_.push_back(lmark_cv_point);
+        
+        eye_left_lmark = EYE_LEFT_INNER_LMARKS[i];
+        landmark = landmarks_.landmark(eye_left_lmark);
+        lmark_cv_point = CvtNormIntoCvPoint(landmark);
+        eye_left_contour_.push_back(lmark_cv_point);
+    }
+}
+
+void EyeAnalyzer::UpdateEyebrow() {    
+    eyebrow_anchor_dist_sum_ = CalculateEyebrowActivity();
+}
+
+double EyeAnalyzer::CalculateEyebrowActivity() {
```

Comment:
Here 2 methods were used for updating a feature value. In others analyzers such as `MouthAnalyzer`, we used only one (`void CalculateFoo()`). We may discuss which approach to use, and from now on, stick to it.

Reply:
I guess keep just the `void Calculate...()` is a better alternative 'cause prevent the existence non informative methods that can make code reading difficult